src/TransactionController.ts

```diff
@@ -129,6 +138,48 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
 		});
 	}
 
+	/**
+	 * Normalizes the transaction information from etherscan
+	 * to be compatible with the TransactionMeta interface
+	 *
+	 * @param txMeta - Object containing the transaction information
+	 * @param currentNetworkID - string representing the current network id
+	 * @returns - TransactionMeta
+	 */
+	private normalizeTxFromEtherscan(txMeta: any, currentNetworkID: string): TransactionMeta {
```

Comment:
We should avoid `any` types in non-test code if possible; it may be good to consider making an interface for an EtherScan response type.